samples/azure/storage-persistence/ASP_2/StorageReader/AzureHelper.cs

```diff
@@ -4,21 +4,22 @@
 using Microsoft.WindowsAzure.Storage;
 using Microsoft.WindowsAzure.Storage.Table;
 using NUnit.Framework;
+using Microsoft.WindowsAzure.Storage.Blob;
 
 [TestFixture]
 public class AzureHelper
 {
     [Test]
     [Explicit]
-    public Task WriteOutData()
+    public async Task WriteOutData()
     {
         #region UsingHelpers
 
-        WriteOutTable("OrderSagaData", false);
-        WriteOutTable("Subscription", true);
-        WriteOutTable("TimeoutDataTableName", false);
-        WriteOutTable("TimeoutManagerDataTable", false);
-        return WriteOutBlobContainer("timeoutstate");
+        await WriteOutTable("OrderSagaData", false);
```

Comment:
I think this requires `ConfigureAwait(false)` we apply that to the doco snippets

Reply:
addressed